tools/sdk-tools/adpcm/util.c

```diff
@@ -10,12 +10,12 @@ static u32 getshort(FILE *ifile)
     u32 c1;
     u32 c2;
 
-    if ((c1 = getc(ifile)) == -1)
+    if ((c1 = getc(ifile)) == -1U)
```

Comment:
Might be better to use `EOF` instead of -1.

Reply:
That should be easy enough to fix. EOF itself should allow for that in place of -1U if I'm not mistaken here

Reply:
Resolved.